Compose readme 2 (#336)
KasperskyLab/Kaspresso: README.md

So, you will not see any difference between tests for View screens and for Compose screens.
More information is available [here](/wiki/10_Jetpack-Compose.md).

> **Be aware of it's an early access that may contain bugs. Also, API can be changed, but we are going to avoid it. Please, be free to create relative issues if you've encountered with any kind of problem.**

Reviewer: ```suggestion
**Be aware of its early access that may contain bugs. Also, API can be changed, but we are going to avoid it. Be free to create relative issues if you've encountered any kind of problem.**
```
